en/editor/assets/index.md

```diff
@@ -1,96 +1,114 @@
+  The **Explorer Panel** is an important tool we use to access and manage project assets. When starting a game, **importing assets** is usually a necessary step. You can use the template project when creating a new project. The project will be opened automatically after the new step is completed. The default layout includes the **Explorer** panel, which contains two asset libraries, referred to as DB, `assets` and `internal`. `internal` belongs to the default built-in assets, which can be copied, but cannot be modified directly.
 
-  ![Panel Overview](img/thumb.gif)
+  Panel operation preview:
 
-## Explorer Panel introduction
+  ![Panel operation preview](img/thumb.gif)
 
-The **Explorer** panel is the main panel. It has a *heads-up menu area*, *a central tree list area*, and a *bottom information display area*.
+## Panel Introduction
 
-- The functions of the *heads-up menu area* are: *New asset button*, *Sort by button*, *Search type button*, *Search box*, *Fold or expand all button*, and *Refresh list button*
-- The *central tree list area* mainly reflects the hierarchical relationship of assets. The root node is the asset root folder corresponding to the file manager of the operating system. The editor calls an __asset database__ *(DB for short)*, and is located in the project folder. It belongs to the project asset; __Internal DB__ comes with the editor and provides some built-in necessary assets. It is a **read-only** asset. It cannot be *added*, *deleted*, *modified*, but **can** be *directly referenced* or *copied*.
-- The *bottom display area* is the URL of the asset after the asset is selected, such as `db://assets`
-- The __right-click__ menu of the panel is also an important function. The contents of the __right-click__ menu in different locations are different, and the unavailable menus are grayed out.
-- The shortcut of the panel currently supports the operation of assets:
+The **Explorer** contains the `head menu area` and the `tree list area` on the panel:
+
+- The functions of the `head menu area` are: **New Resource Button**, **Sorting Method Button**, **Search Filter**, **Search Box**, **All Collapse or Expand Button** , **Refresh list button**
+- The `tree list area` mainly reflects the relationship between assets. The root node such as `assets` is similar to the file manager in the operating system. The editor is called a DB.
+   * `Assets` is a project asset, which is empty by default in a new project;
+   * `Internal` is a built-in asset. It is a read-only asset and cannot be added, deleted or modified, but it can be used as a asset template and copied to Assets DB for pasting, that is, a new project asset is created.
```

Comment:
`Internal` is the built-in asset mount. It is a read-only asset and cannot be added, deleted or modified, but it can be directly referenced by your project. You can also copy any internal asset to the project `assets` mount as a template, then modify it as you wish.